src/main/java/uk/gov/hmcts/reform/wacaseeventhandler/services/CaseEventMessageMapper.java

```diff
@@ -39,16 +39,19 @@ public CaseEventMessage mapToCaseEventMessage(CaseEventMessageEntity entity) {
             entity.getRetryCount());
     }
 
+    @SuppressWarnings("PMD.ConfusingTernary")
     private String getCaseTypeId(CaseEventMessageEntity entity) {
         String caseTypeId = null;
-        if (entity.getMessageContent() != null) {
+        if (entity.getMessageContent() != null && !entity.getMessageContent().isBlank()) {
```

Comment:
New condition. As agreed, we should cover new logic with the unit test. Can we add a unit test for this scenario